packages/apps/plugins/plugin-table/src/components/TableSettings/TableSettings.stories.tsx

```diff
@@ -8,6 +8,8 @@ import React, { useEffect, useState } from 'react';
 
 import { TableType } from '@braneframe/types';
 import { createSpaceObjectGenerator } from '@dxos/echo-generator';
+import * as E from '@dxos/echo-schema';
```

Comment:
```suggestion
import { create, type DynamicEchoSchema } from '@dxos/echo-schema';
```

we're trying to move away from the `import * as E` syntax